src/sourcemap-register.js

```diff
@@ -0,0 +1,3 @@
+// note: this file is overriden by `scripts/build`
+// and substituted for the production release
+module.exports = require("source-map-support/register");
```

Comment:
This makes it work both inside `dist/` and in `src/`

Reply:
Ah, thanks for sorting out this use case too.